modules/drivers/xml/xerces/CMakeLists.txt

```diff
@@ -1,306 +1,313 @@
-set(TARGET_NAME xerces-c)

```

Comment:
Looks like line endings got flipped on this file, can you flip them back

Reply:
Line endings in this file were previously flipped from LF to CRLF in bc5552cd209eb937da82dc433aafed9eb89be1ca. This change restores the LF line endings. You can hide the whitespace changes in the Github diff: https://github.blog/2018-05-01-ignore-white-space-in-code-review/

I also added a .gitattributes file marking CMakeLists.txt (as well as for *.in, *.cmake, and conanfile.py) as `text` files, which turns on text line ending normalization in git (https://git-scm.com/docs/gitattributes). This sets git to use LF endings internally for these files, which should prevent this from happening again.

By default, git uses LF line endings internally for text files but converts to CRLF on checkout for Windows. I set an additional option in .gitattributes to force LF on checkout for all platforms for those 4 file types (`eol=lf`). This is needed to support conan recipe revisions. When a conan recipe is exported, the conanfile.py and any files listed in `exports_sources` are hashed to create a recipe revision id. Line ending differences change the hash result, which would cause the recipe ID to change if you upload a package built on Windows to a remote (such as an Artifactory server) and subsequently upload a package built on Linux to the same remote. These recipe revision ID changes due to whitespace would require an unnecessary rebuild of any packages built with the previous recipe revision ID.

Reply:
👍